config/config_util.go

```diff
@@ -155,11 +155,25 @@ func flatten(flatMap map[string]interface{}, nested interface{}, prefix string)
 	}
 }
 
-// GetTxnScopeFromConfig extract txn_scope default value from config
+const (
+	globalTxnScope = "global"
+	localTxnScope  = "local"
+)
+
+// GetTxnScopeDefaultValueFromConfig extract default txn_scope value from config
+func GetTxnScopeDefaultValueFromConfig() string {
```

Comment:
Now there are 2 `txnScope` in the project: 
- one is used in the oracle, whose value is the zone name or "global".
- one is used in the session variable, whose value is "local" or "global".
This may make other developers confused. Any ideas?

Reply:
A tuple of `setting scope type indicator, real txn scope string` can combine these two into one, just like an `Maybe<Scope>` in cpp, or `enum { Global, Local(string) }` in rust.

Reply:
I'd like to make `sessionVar.TxnScope` as a struct which holds 2 attributes: 1. the value of @@txn_scope. 2. the value of dcLocation.

In this way, I think we can avoid make developers confused and also don't have too much work to do.